src/ibl_to_nwb/_scripts/convert_brainwide_map_raw_only.py

```diff
@@ -0,0 +1,84 @@
+from pathlib import Path
+
+from neuroconv.datainterfaces import SpikeGLXRecordingInterface
+from one.api import ONE
+
+from ibl_to_nwb.converters import BrainwideMapConverter
+from ibl_to_nwb.datainterfaces import RawVideoInterface
+
+session_id = "d32876dd-8303-4720-8e7e-20678dc2fd71"
+
+# Specify the revision of the pose estimation data
+# Setting to 'None' will use whatever the latest released revision is
+revision = None
+
+base_path = Path("E:/IBL")
+base_path.mkdir(exist_ok=True)
+nwbfiles_folder_path = base_path / "nwbfiles"
+nwbfiles_folder_path.mkdir(exist_ok=True)
+
+# Initialize IBL (ONE) client to download processed data for this session
+one_cache_folder_path = base_path / "cache"
+ibl_client = ONE(
+    base_url="https://openalyx.internationalbrainlab.org",
+    password="international",
+    silent=True,
+    cache_dir=one_cache_folder_path,
+)
+
+# Specify the path to the SpikeGLX files on the server
+probe_1_source_folder_path = Path("D:/example_data/ephy_testing_data/spikeglx/Noise4Sam_g0")
+probe_2_source_folder_path = Path(
+    "D:/example_data/ephy_testing_data/spikeglx/multi_trigger_multi_gate/SpikeGLX/5-19-2022-CI0/5-19-2022-CI0_g0/"
+)
+
+ap_1_file_path = probe_1_source_folder_path / "Noise4Sam_g0_imec0/Noise4Sam_g0_t0.imec0.ap.bin"
+ap_2_file_path = probe_2_source_folder_path / "5-19-2022-CI0_g0_imec0/5-19-2022-CI0_g0_t0.imec0.ap.bin"
+
+lf_1_file_path = probe_1_source_folder_path / "Noise4Sam_g0_imec0/Noise4Sam_g0_t0.imec0.lf.bin"
+lf_2_file_path = probe_2_source_folder_path / "5-19-2022-CI0_g0_imec0/5-19-2022-CI0_g0_t0.imec0.lf.bin"
```

Comment:
@grg2rsr I'm just emulating what I can with local SpikeGLX files here. Adjust as needed for your actual conversion